Cargo.lock

```diff
@@ -946,6 +955,7 @@ dependencies = [
  "autocfg",
  "cc",
  "libc",
+ "openssl-src",
```

Comment:
All the changes in this file seem spurious leftover from previous development iteration.